ibridges/interactive.py

```diff
@@ -81,14 +84,22 @@ def interactive_auth(
     raise LoginError("Connection to iRODS could not be established.")
 
 
-def _from_pw_file(irods_env_path, **kwargs):
+def _from_pw_file(irods_env_path, irodsa_backup: Optional[str] = None, **kwargs):
     try:
         session = Session(irods_env_path, **kwargs)
         return session
     except IndexError:
         print("INFO: The cached password in ~/.irods/.irodsA has been corrupted")
     except PasswordError:
-        print("INFO: The cached password in ~/.irods/.irodsA is wrong.")
+        print(irodsa_backup)
```

Comment:
I think this print statement needs to be deleted, otherwise we are not conforming to security requirements ;)